.github/workflows/deploy.yaml

```
@@ -36,3 +35,14 @@ jobs:
           # You can swap them out with your own user credentials.
           user_name: github-actions[bot]
           user_email: 41898282+github-actions[bot]@users.noreply.github.com 
+      - name: Check New Version
+        uses: tj-actions/changed-files@23.1
+        id: check-new-version
+        with:
+          files: versions.json
+      - name: Trigger Algolia Crawler
+        if: steps.check-new-version.any_changed == 'true'
```

I think we should trigger this after every deploy to reflect any changes under docs/ and versioned_docs/.
```suggestion
      - name: Trigger Algolia Crawler
```

---

We can try this. Something in the documentation made me concerned about hitting a rate limit crawl requests, but after reviewing I can't seem to find any such indication. Something to keep an eye on though.